src/DependencyInjection/Compiler/KernelPass.php

```diff
@@ -28,13 +28,17 @@ public function process(ContainerBuilder $container)
         }
         $container->removeAlias('ezpublish.http_cache.purger');
         $arguments = $container->getDefinition('cache_clearer')->getArguments();
-        $arguments[0] =  array_values(array_filter($arguments[0], function ($argument) {
-            if ($this->isCachePurger($argument)) {
-                return false;
-            }
-            return true;
-        }));
-        $container->getDefinition('cache_clearer')->setArguments($arguments);
+
+        // BC Symfony < 3.4
+        if (is_array($arguments[0])) {
```

Comment:
Ok, but let's keep early return convention, so
```php
if (!is_array($arguments[0])) {
    return;
}
```

Reply:
nvm :)

Reply:
@alongosz could you review e5f73817ca328669735b279a76ada487b6c03409

Reply:
e5f7381 approved ;)